modules/ROOT/pages/dataweave-variables.adoc

```diff
@@ -151,6 +151,67 @@ Note that DataWeave variables cannot be reassigned. They are also distinct from
 
 You can combine several local variable definitions as a comma separated list inside the using function. For example: `using (firstName='Annie', lastName='Point')`
 
+== Naming Rules for Variables
+
+Variable names have certain requirements to be valid, following are the rules for naming variables:
```

Comment:
grammar issue with comma there. Suggest something like "Variable names must meet the following requirements to be valid:" or something similar.